tests/YGDefaultValuesTest.cpp

```diff
@@ -25,9 +25,9 @@ TEST(YogaTest, assert_default_values) {
   ASSERT_EQ(YGPositionTypeRelative, YGNodeStyleGetPositionType(root));
   ASSERT_EQ(YGWrapNoWrap, YGNodeStyleGetFlexWrap(root));
   ASSERT_EQ(YGOverflowVisible, YGNodeStyleGetOverflow(root));
-  ASSERT_FLOAT_EQ(0, YGNodeStyleGetFlexGrow(root));
-  ASSERT_FLOAT_EQ(0, YGNodeStyleGetFlexShrink(root));
-  ASSERT_FALSE(YGNodeStyleGetFlexBasis(root).unit != YGUnitAuto);
+  ASSERT_TRUE(YGFloatIsUndefined(YGNodeStyleGetFlexGrow(root)));
```

Comment:
An alternative is, to return 0 for these values in a (custom) getter, and keep the undefined as an internal implementation detail?